accepted/00076-configuration-channels-visibility.md

```diff
@@ -0,0 +1,201 @@
+This page represents the assignment connection between a system group and configuration channels. All the systems in this system group will inherit the selection set of channels here. This should be only a list of Configuration Channels with the possibility of simply select/deselect channels and Apply changes. If there is a need to show only the channel selected, a filter to the list page should be added instead.
+
+Before:
+![System Groups BEFORE](images/configuration-channels-visibility/Systems_SystemGroups_Details_States_ConfigurationChannels_BEFORE.png)
+
+After:
+![System Groups AFTER](images/configuration-channels-visibility/Systems_SystemGroups_Details_States_ConfigurationChannels_AFTER.png)
+
+### `Systems > System Set Manager > Overview > Configuration > Subscribe to Channels`
+Before:
+![SSM Subscribe To Channels BEFORE](images/configuration-channels-visibility/Systems_SSM_Configuration_SubscribeToChannels_BEFORE.png)
+
+After:
+![SSM Subscribe To Channels AFTER](images/configuration-channels-visibility/Systems_SSM_Configuration_SubscribeToChannels_AFTER.png)
+
+### `Systems > System Set Manager > Overview > Configuration > Unsubscribe to Channels`
+Before:
+![SSM Unsubscribe From Channels BEFORE](images/configuration-channels-visibility/Systems_SSM_Configuration_UnsbscribeFromChannels_BEFORE.png)
+
+After:
+![SSM Unsubscribe From Channels AFTER](images/configuration-channels-visibility/Systems_SSM_Configuration_UnsubscribeFromChannels_AFTER.png)
+
+### `Configuration > Overview`
+Before:
+![Configuration Overview BEFORE](images/configuration-channels-visibility/Configuration_Overview_BEFORE.png)
+
+After:
+![Configuration Overview AFTER](images/configuration-channels-visibility/Configuration_Overview_AFTER.png)
+
+### `Configuration > Channels`
+Before:
+![Configuration Channels BEFORE](images/configuration-channels-visibility/Configuration_Channels_BEFORE.png)
+
+After:
+![Configuration Channels AFTER](images/configuration-channels-visibility/Configuration_Channels_AFTER.png)
+
+### `Configuration > Channels > Details > Overview`
+Before:
+![Configuration Channels Detail Overview BEFORE](images/configuration-channels-visibility/Configuration_Channels_Details_Overview_BEFORE.png)
+
+After:
+![Configuration Channels Detail Overview AFTER](images/configuration-channels-visibility/Configuration_Channels_Details_Overview_AFTER.png)
+
+### `Configuration > Channels > Details > Systems > Inheriting Systems`
+This is a new page where systems are listed only if not subscribed to the channel directly, but inheriting it through Organization or System Group assignment.
+
+Before:
+![Configuration Channels Detail Systems BEFORE](images/configuration-channels-visibility/Configuration_Channels_Details_Systems_BEFORE.png)
+
+After:
+![Configuration Channels Detail Systems AFTER](images/configuration-channels-visibility/Configuration_Channels_Details_Systems_AFTER.png)
+
+
+### `Configuration > Channels > Details > System Groups`
```

Comment:
What would be the plan for implementation? Write in it ReactJS? Or generalizing the struts one somehow?